stubs/aio_pika/__init__.pyi

```diff
@@ -0,0 +1,46 @@
+from aio_pika import patterns, pool
```

Comment:
we could also try updating to `aio-pika` 7 which has typing built-in? https://github.com/mosquito/aio-pika/blob/master/CHANGELOG.md

Reply:
I'm a bit reluctant to do this now, as this would need more work and more testing. But I'll add a FIXME to mention the update